lib/livingstyleguide/document.rb

```diff
@@ -84,10 +84,10 @@ def erb
   end
 
   def evaluate(scope, locals, &block)
+    scripts_path = "#{File.dirname(__FILE__)}/templates/scripts"
     @head = ""
-    @head << ERB.new(File.read("#{File.dirname(__FILE__)}/templates/javascript-copy.html.erb")).result(binding)
-    @head << ERB.new(File.read("#{File.dirname(__FILE__)}/templates/code-example.html.erb")).result(binding)
-    @head << ERB.new(File.read("#{File.dirname(__FILE__)}/templates/colors-example.html.erb")).result(binding)
+    @javascript = ""
+    @javascript << ERB.new(File.read("#{File.dirname(__FILE__)}/templates/scripts.html.erb")).result(binding)
```

Comment:
Just directly read and add the files here. No need to call an ERB template for this as there’s nothing else than Ruby code.